apps/ejabberd/src/mod_mam_cassandra_arch.erl

```diff
@@ -501,13 +492,14 @@ from_id(ID, Filter = #mam_ca_filter{start_id = AfterID}) ->
 rows_to_uniform_format(MessageRows) ->
     [row_to_uniform_format(Row) || Row <- MessageRows].
 
-row_to_uniform_format([MessID, BSrcJID, Data]) ->
-    SrcJID = unserialize_jid(BSrcJID),
-    Packet = stored_binary_to_packet(Data),
-    {MessID, SrcJID, Packet}.
+row_to_uniform_format(#{from_jid := FromJID, message := Msg, id := Id}) ->
+    SrcJID = unserialize_jid(FromJID),
+    Packet = stored_binary_to_packet(Msg),
+    MsgID = Id,
```

Comment:
Do we need `MsgID`? Can we simply return `Id` or assign `id` from the input map to `MsgID`?